javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (studentName != "") {
+    if (!class07Students.includes(studentName)) {
+      if (studentName === "Queen") class07Students.push(studentName);
+      else {
+        if (class07Students.length <= 5) class07Students.push(studentName);
+        else console.log("Cannot add more students to class 07");
+      }
+    } else console.log(`Student ${studentName} is already in the class`);
+  } else console.log("Student name can not be empty");
+}
```

Comment:
you could avoid some of the nested statements by using logical operators. That makes your code faster to run and easier to read.

Reply:
Even though you are right but I think according to the problem description and outcomes I do not think we can use logical operators here.